.changeset/famous-brooms-hunt.md

```diff
@@ -1,6 +1,6 @@
 ---
-"@steveojs/scheduler-sequelize": major
-"@steveojs/scheduler-prisma": major
+"@steveojs/scheduler-sequelize": minor
+"@steveojs/scheduler-prisma": minor
```

Comment:
this was supposed to be minor, v7 is the major change where we switched to the new library. this change now becomes v7.1.0

Reply:
This is technically an API breaking change.
However, if no one (apart from the monolith) has been using 7.x.x with reference to these functions then I am okay with it.

Reply:
@andersonku no other references to 7.x